test/scripts/start-services.sh

```diff
@@ -62,32 +88,16 @@ start_parachain()
 
     echo "Writing Polkadot configuration"
     cp config.json $configdir/polkadotConfig.json
-    parachain_conf="{
-        \"bin\": \"$bin\",
-        \"id\": \"200\",
-        \"wsPort\": 11144,
-        \"port\": 31200,
-        \"flags\": [
-            \"--execution=native\",
-            \"-lruntime=debug,import_header=trace,bridge=trace\",
-            \"--rpc-cors=all\",
-            \"--offchain-worker=Always\",
-            \"--enable-offchain-indexing=true\",
-            \"--\",
-            \"--execution=wasm\"
-        ],
-        \"chain\": \"$configdir/spec.json\"
```

Comment:
This line is important. Will break if not overriden.

Reply:
has been fixed